### DISCUSSION TITLE
Eleccion de proyecto

### DISCUSSION THREAD

**Discussion by manuelaarteaga** [rramosp/ai4eng.v1#275] [00 - General]
¡Hola profe!
Mi grupo ha estado buscando datasets de kaggle para el proyecto, y me gustaría preguntar si es posible utilizar este https://www.kaggle.com/datasets/einsteindata4u/covid19?resource=download para predecir el ingreso a sala general, unidad semiintensiva o unidad de cuidados intensivos entre los casos confirmados de COVID-19.

**Comment by Felipe-RA** [comment]
Saludos @manuelaarteaga 

Como siempre, recordarte que cuando decides elegir de la categoría datasets, y no de la de competiciones, tomas el reto extra de tener que definir tú la tarea predictiva y las métricas de desempeño (así como justificar el por qué de ambas cosas) 

**Este dataset tiene la particularidad de que está planteado como una competición no formal**, y te proponen dos tareas predictivas. No te dan una métrica estricta, así que la tendrás que definir y justificar.

En cuanto al dataset como tal, tiene la ventaja de que tiene mucha discusión y código de terceros, lo que te puede servir para orientarte durante el proceso.

Cumple los requisitos de dimensionalidad, tanto en filas como en columnasP.

Puedo observar que tiene ~110 columnas y a ojo conté más de 12 columnas categóricas, así que cumple también el requisito del 10% como columnas categóricas. También cumple los datos faltantes.

Todo en orden para que comiencen a trabajar, estamos pendientes de cualquier duda extra!
